cmd/manager/main.go

```diff
@@ -149,11 +214,14 @@ func main() {
 		}
 	}()
 
+	secretsChan := make(chan *corev1.Secret)
+
 	controllerConfigManager := controllers.KubeEnvoyConfigManager{
 		Client:       mgr.GetClient(),
 		Scheme:       mgr.GetScheme(),
 		EnvoyManager: envoyManager,
 		Validator:    proxy,
+		SecretsChan:  secretsChan,
```

Comment:
I think we need to name it differently, so readers would understand what this field does. SecretsChan is too generic. I propose "WatchedSecretsChan".